test/widgets/action_sheet_test.dart

```diff
@@ -38,11 +38,12 @@ late FakeApiConnection connection;
 Future<void> setupToMessageActionSheet(WidgetTester tester, {
   required Message message,
   required Narrow narrow,
+  Account? account,
 }) async {
   addTearDown(testBinding.reset);
-
-  await testBinding.globalStore.add(eg.selfAccount, eg.initialSnapshot());
-  final store = await testBinding.globalStore.perAccount(eg.selfAccount.id);
+  final effectiveAccount = account ?? eg.selfAccount;
+  await testBinding.globalStore.add(effectiveAccount, eg.initialSnapshot());
```

Comment:
We probably want to override `zulipFeatureLevel` to `account.zulipFeatureLevel` for `eg.initialSnapshot` too.